### ISSUE TITLE
Feature request: address bar at the bottom

### ISSUE THREAD

**Issue by zsbolyoczki** [brave/brave-browser#12883]
Issue #7511 seems to be closed still I can't find this option in  v1.17.72 on Android.

As it is reasoned in the above mentioned ticket the lack of this feature makes Brave uncomfortable on big screen devices (even on devices with 5"+ screen) and putting the address bar to the top is not practical on mobile devices.

Because not all the users like to have address bar at the bottom there should be an option for this in the Settings.

**Comment by DominicDesbiens** [comment]
I agree with this. Don't know why previous issue was closed. There is toolbar at bottom but no option to move the address bar at bottom. Many Android browsers offer this option right now. I prefer to have address bar at bottom !

**Comment by sheikh-azharuddin** [comment]
Vivaldi and Firefox already has the option..please add the same

**Comment by jawadalkassim** [comment]
> Vivaldi and Firefox already has the option..please add the same

Kiwi also have the same feature and the browser is fully open source.

**Comment by kevinkosterr** [comment]
I need this. The stretch of my fingers to the top is horrible.

**Comment by ahmuro** [comment]
Every once in a while I check Brave again to see if I can have the URL bar at the bottom of the screen yet. It has been 3 years already.

**Comment by Resaltare** [comment]
Still nothing... will check back in 2030...maybe

**Comment by Saoiray** [comment]
@bsclifton wanted to tag you in on this as I know it still comes up on occasion. Is this something that has to be done upstream or is it a change Brave would be willing and able to do?

**Comment by Resaltare** [comment]
> Any updates on this issue?

Yes, there are called Vivaldi, Kiwi browser and whatnot.

**Comment by fn5** [comment]
I would really like this option. I have hand issues and this would be a good thing to choose. Want to support and make use of Brave at the same time (:

**Comment by vanhoopstallion** [comment]
2024 Q2 check in

**Comment by deeeseee** [comment]
This is the single and only reason why I will not use Brave as my browser on Android. It's too uncomfortable to try to stretch my thumb towards the top of the screen, and risks dropping the phone. So I wholeheartedly support providing this feature.

**Comment by dvux1988** [comment]
Miss this feature, too.

**Comment by mailinglists35** [comment]
what is the block on doing this? is it a technical reason? is it a brave policy reason (aka we simply don't want to)? something else?

huge phones are mainstream so reaching the addressbar single handed is effectively impossible on 99% of phones

**Comment by mailinglists35** [comment]
@hffvld @brave-builds

**Comment by bsclifton** [comment]
@mailinglists35 upstream Chromium is already working on this (not done yet). When it's ready, we'll have that code too and we'll reconcile how it works with our UI. It'll likely start as behind a flag and then we'll let folks choose

**Comment by vanhoopstallion** [comment]
hallelujah! Finally.

Can't wait to start using brave on Android at last

**Comment by fallaciousreasoning** [comment]
@vanhoopstallion I'm afraid that's from last time they added the feature :/

**Comment by vanhoopstallion** [comment]
> [@vanhoopstallion](https://github.com/vanhoopstallion) I'm afraid that's from last time they added the feature :/

It's in Chrome Canary build on playstore. I've checked. Follow the flag setting and try it out

**Comment by vanhoopstallion** [comment]
It's live via flag switch.

![Image](https://github.com/user-attachments/assets/3186b8c8-0a7a-41d9-bd3a-0ed99410cea4)

**Comment by vanhoopstallion** [comment]
chrome://flags/#android-bottom-toolbar

Enable, exit to restart. Then enable in settings.

**Comment by vanhoopstallion** [comment]
Beta brave is getting there. Stable is good enough for me to switch over desktop and mobile. Hello brave it's been a while

**Comment by fallaciousreasoning** [comment]
Also doesn't play well with tab groups

**Comment by mailinglists35** [comment]
doesn't work on Samsung S23 Android 14 latest brave release from store.

**Comment by Dominic-t3ch** [comment]
> Also doesn't play well with tab groups

Yes same here on my Pixel 7 Pro, Android 15. The bottom address bar works, just waiting that the problem with grouped tab is fixed to use the bottom bar (it show over the grouped tabs bar).

**Comment by fallaciousreasoning** [comment]
To get tab groups working with it you need to set (in chrome://flags)

1. #android-bottom-toolbar
2. Enable #bottom-browser-controls-refactor “Enabled Dispatch yOffset” option

**Comment by Dominic-t3ch** [comment]
> To get tab groups working with it you need to set (in chrome://flags)
> 
>     1. #android-bottom-toolbar
> 
>     2. Enable #bottom-browser-controls-refactor “Enabled Dispatch yOffset” option
> 
>     3. Restart the browser (might need to do this twice)
> 
>     4. Long press the address bar
> 
>     5. Move to bottom
> 
> 
> See also https://issues.chromium.org/u/2/issues/390936402#comment5

Hey, the Dispatch yOffset works !! Just be sure to disable the bottom toolbar in Appearence settings. If enabled you lost access to menu and tabs icon. Thank you !

**Comment by ElectricBoot** [comment]
There is another issue with the current implementation: In the new tab, the address bar is still at the top. The address bar comes down to bottom only after going to a webpage. Brave 1.77.97, Android 14.

**Comment by mailinglists35** [comment]
can we have the buttons bar and url bar swapped, please? it's awkward how it is after flags enabing. 

![Screenshot_20250515_191352_Brave.png](https://github.com/user-attachments/assets/82fa4b05-849f-40a5-8296-e46c1198dbdd)

**Comment by bsclifton** [comment]
Closing as duplicate of https://github.com/brave/brave-browser/issues/7356

This is possible already if you visit brave://flags and search for `android-bottom-toolbar` flag and enable. We'll be rolling this out when the time is right - subscribe to https://github.com/brave/brave-browser/issues/7356 for more information.
